certified-connectors/Payables E2E/apiDefinition.swagger.json

```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "IN-D Invoice Data Capture",
+    "description": "A smart invoice processing solution that converts static documents (scanned, PDFs, screenshots, etc.) into usable information to fuel your business processes.IN-D Payables is an AI led intelligent document classification and capture solution that automates processing of all kinds of commercial documents used in Shipping, Logistics and Manufacturing operations. Manual handling of documents slows the pace of business transactions."
```

Comment:
@gayathri35 please make sure that everything is grammatically correct and is free of typographical errors. 
Connector description should describe concisely the main purpose and value offered by your connector.